src/middlewared/middlewared/etc_files/ganesha/ganesha.conf.mako

```diff
@@ -4,15 +4,19 @@
     config = middleware.call_sync("nfs.config")
 
     shares = middleware.call_sync("sharing.nfs.query", [["enabled", "=", True]])
+    if not shares:
+        raise FileShouldNotExist()
 
     kerberos_keytabs = middleware.call_sync("kerberos.keytab.query")
 
     gc = middleware.call_sync("network.configuration.config")
 
     # call this here so that we don't have to call this
     # n times (n being the number of shares in db)
-    peer_job = middleware.call_sync("gluster.peer.status")
-    peers = peer_job.wait_sync()
+    peers = []
+    if middleware.call_sync("service.started", "glusterd"):
+        peer_job = middleware.call_sync("gluster.peer.status")
+        peers = peer_job.wait_sync()
```

Comment:
Is it supposed to always return fast? We are waiting for a job in a non-job.

Reply:
Just reminded me this isn't a job anymore with recent changes.